package.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "bs-node",
+    "version": "0.0.1",
+    "description": "BuckleScript bindings to the Node.js API",
+    "scripts": {
+        "start": "bsb -make-world -w",
+        "build": "bsb -make-world",
+        "test": "npm run build && jest",
+        "watch:bsb": "bsb -make-world -w",
+        "watch:jest": "jest --watchAll",
+        "watch:screen": "screen -c .screenrc"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/BuckleTypes/bs-jest.git"
+    },
+    "keywords": [
+        "node",
+        "bucklescript"
+    ],
+    "author": "",
+    "license": "MIT",
+    "bugs": {
+        "url": "https://github.com/BuckleTypes/bs-node/issues"
+    },
+    "homepage": "https://github.com/BuckleTypes/bs-node#readme",
+    "devDependencies": {
+        "bs-platform": "1.5.0",
```

Comment:
You probably want to track bs master here, to keep up with all the good stuff coming.